src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
```

Comment:
To show properly in Java help, for all languages, it should be defined the charset here (`<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">`).